### ISSUE TITLE
Screen reader accessibility

### ISSUE THREAD

**Issue by steverep** [NREL/SAM#2123]
### Describe the bug

Many of SAM's UI elements fail basic accessibility.  For example, many buttons do not have an accessible name, many input fields are not associated with their on-screen accessible name, tab behavior is not intuitive in some areas and doesn't work at all in others, and important non-input screens like the simulation results output are not accessible at all.

### To Reproduce

I would suggest downloading a free screen reader like [NVDA](https://www.nvaccess.org/) and try to navigate SAM using a keyboard.  The difficulties should be apparent right away but let me know if further instructions would help.

### Expected behavior

At a minimum, the screen reader should announce a label for all input fields, buttons, and menu items.  Other text on screen should be easily navigable.

### Operating System and Version

 - OS: Windows 10
 - Version: 2025.4.16

### Additional context

I only tested on Windows with NVDA, but the issue is likely with any OS or screen reader.

**Comment by janinefreeman** [comment]
Hi @steverep - thank you for bringing this to our attention. We will add this to our list of upgrades to make and consult with our IT department and project sponsors about the best way to move these fixes forward.
